SPEC.md

```diff
@@ -3358,7 +3358,7 @@ Although a WDL workflow and the task(s) it calls may be defined completely withi
 
 The `import` statement is the basis for modularity in WDL. A WDL document may have any number of `import` statements, each of which references another WDL document and allows access to that document's top-level members (`task`s, `workflow`s, and `struct`s).
 
-The `import` statement specifies a WDL document source as a string literal, which is interpreted as a URI. The execution engine is responsible for resolving each import URI and retrieving the contents of the WDL document. The contents of the document in each URI must be WDL source code **of the same version as the importing document**.
+The `import` statement specifies a WDL document source as a string literal, which is interpreted as a URI. The execution engine is responsible for resolving each import URI and retrieving the contents of the WDL document. The contents of the document in each URI must be a WDL document **with the same major version and a minor version less than or equal to the minor version of the importing document**.
```

Comment:
I think there should be a mention of the effective version of the wdl document. Ie all imported docs are treated as the version of the first imported document.

Reply:
I can definitely point it out though, I don't know if this is a _requirement_ rather than a liberty than an execution engine might take to make their lives easier. If, for example, an execution engine wants to parse the various versions individually according to the defined version in the document and then stitch them together internally using its abstract representation, I see no issue with that either.